UnitsNet.Tests/CustomCode/SpeedTests.cs

```diff
@@ -68,6 +68,12 @@ public class SpeedTests : SpeedTestsBase
 
         protected override double MillimetersPerHourInOneMeterPerSecond => 3600000;
 
+        protected override double InchesPerSecondInOneMeterPerSecond => 39.3700787;
```

Comment:
I think you've mixed two pieces of work here. I saw you were working on the master branch, but it would probably be better for you to work on a feature branch to separate the two bulks of changes and create separate pull requests.
We can proceed like this if you are not familiar with how to do it, no biggie, but just a heads up for next time.